CONTRIBUTING.md

```diff
@@ -66,11 +66,19 @@ compromise among collaborators be the default resolution mechanism.
 
 # Becoming a Member and Collaborator
 
-To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, investigate issues tagged as `good first issue`,
+To become a Member of the Community Committee, start participating! Attend scheduled meetings, investigate issues tagged as `good first issue`,
 file issues and pull requests, and provide insight via GitHub. Request to become an Observer by
 filing an issue. Once added as an Observer to meetings, we will track attendance and participation
-for 3 months, as according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee). If you meet the minimum
-attendance and are participating, the CommComm will vote to add you as a member.
+according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee).
+Once you meet the minimum attendance and participation requirements of a Member, you will automatically become a Member.
+To help expedite the process, you are encouraged to open a PR adding yourself, but it is not a requirement. The Chair is
```

Comment:
I see TSC operating in a way that nominations are made someone other than the Member. I think that is a much better model. I know we are a bit under resource in the CommComm right now, so this speaks to reality. Encouraging a qualified member to submit an issue and then building a process where an existing Member would create the PR with the proof would be a better model.

Reply:
@dshaw while that is historical, my understanding as written is that you -can- nominate yourself under the current TSC governance, it's just that no one has done it.

Reply:
See: https://github.com/nodejs/TSC/issues/205